src/misc/parse_arguments.cpp

```diff
@@ -119,7 +119,7 @@ argparse::ArgumentParser parsed_arguments(
     .help(pre + "IDs of geographic divisions [default: 1st CSV column header]");
   arguments.add_argument("-A", "--area")
     .help(pre + "target areas [default: 2nd CSV column]");
-  arguments.add_argument("-C", "--color")
+  arguments.add_argument("-C", "--color", "--colour")
```

Comment:
This is a good additiona, but it will not solve our british english problem unfortunately.